[2024-05-14T08:02:01.387+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T08:02:01.396+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T08:01:54.539823+00:00 [queued]>
[2024-05-14T08:02:01.399+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T08:01:54.539823+00:00 [queued]>
[2024-05-14T08:02:01.399+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T08:02:01.405+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_if_update> on 2024-05-14 08:01:54.539823+00:00
[2024-05-14T08:02:01.408+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=83) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T08:02:01.408+0000] {standard_task_runner.py:63} INFO - Started process 85 to run task
[2024-05-14T08:02:01.409+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'check_if_update', 'manual__2024-05-14T08:01:54.539823+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_jobs104.py', '--cfg-path', '/tmp/tmph35zuabe']
[2024-05-14T08:02:01.410+0000] {standard_task_runner.py:91} INFO - Job 62: Subtask check_if_update
[2024-05-14T08:02:01.431+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T08:01:54.539823+00:00 [running]> on host ceeaa83f1981
[2024-05-14T08:02:01.469+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='check_if_update' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T08:01:54.539823+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T08:01:54.539823+00:00'
[2024-05-14T08:02:01.470+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T08:02:01.476+0000] {logging_mixin.py:188} INFO - check_if_update !!!
[2024-05-14T08:02:01.476+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T08:02:01.476+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T08:02:01.481+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=check_if_update, run_id=manual__2024-05-14T08:01:54.539823+00:00, execution_date=20240514T080154, start_date=20240514T080201, end_date=20240514T080201
[2024-05-14T08:02:01.509+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T08:02:01.523+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T08:02:01.524+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
